test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -201,6 +201,30 @@ public void execute_list_showsAllPersons() throws Exception {
                               expectedList);
     }
 
+    @Test
+    public void execute_sort_showsAllPerson() throws Exception {
+        // prepare expectations
+        TestDataHelper helper = new TestDataHelper();
+        Person person1 = helper.generatePersonWithName("John Tan");
+        Person person2 = helper.generatePersonWithName("Ali Baba");
+        Person person3 = helper.generatePersonWithName("Jill Tan");
+        Person person4 = helper.generatePersonWithName("Bob Lee");
+
+        List<Person> expectedList = helper.generatePersonList(person2, person4, person3, person1);
+        List<Person> unorderedList = helper.generatePersonList(person1, person2, person3, person4);
+
+        AddressBook expectedAB = helper.generateAddressBook(unorderedList);
```

Comment:
Why is your expectedAB generated from the `unorderedList`? Should it not be from `expectedList`?
You might have meant `unorderedAB` instead?